src/drivers/cmakeDriver.ts

```diff
@@ -868,7 +868,23 @@ export abstract class CMakeDriver implements vscode.Disposable {
 
     private configRunning: boolean = false;
 
-    private buildRunning: boolean = false;
+    private _buildRunning: boolean = false;
+    get buildRunning(): boolean {
+        return this._buildRunning;
+    }
+
+    set buildRunning(running: boolean) {
+        if (!running) {
+            this._requestedTargets = undefined;
+        }
```

Comment:
`this._buildRunning` should be updated in this function

Reply:
`set` and `get` were removed.